packages/headless/src/chat.ts

```diff
@@ -202,12 +237,13 @@ export function useUpdateRoom({
 /**
  * The hook responsible for integration with the wss server, it allows the user to send
  * delete and update messages.
+ * Each Socket Event takes 3 params: the event, the payload, the callback which will be invoked in case of an error.
```

Comment:
The max JS Doc line should be 80 char. You can use the vs-code extension "rewrap" to wrap you comments when exeed 80 char.